packages/opentelemetry-core/package.json

```diff
@@ -88,11 +88,11 @@
     "webpack": "4.46.0"
   },
   "peerDependencies": {
-    "@opentelemetry/api": ">=1.0.0 <1.8.0"
+    "@opentelemetry/api": ">=1.1.0 <1.8.0"
   },
   "dependencies": {
     "@opentelemetry/semantic-conventions": "1.18.1"
   },
   "homepage": "https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-core",
   "sideEffects": false
-}
+}
```

Comment:
Please add a new line at the end of the file.

Reply:
Oh didn't realize my editor dropped that, fixed https://github.com/open-telemetry/opentelemetry-js/pull/4408/commits/6e9e47dc34e1a94b7f3dc5976c383a7fb9773a72